scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx

```diff
@@ -388,10 +399,20 @@ export function getAutoTaggingComponent(superdesk: ISuperdesk, label: string) {
 
                                 const others = items.filter((tag) => isEntity(tag) === false);
                                 const othersGrouped = others.groupBy((tag) => tag.group.value);
-                                const othersGroupedAndSorted = othersGrouped.sortBy(
-                                    (_, key) => this.iMatricsFields[key].order ?? -1,
+
+                                const othersGroupedAndSortedByConfig = othersGrouped
+                                    .filter((_, key) => !!this.iMatricsFields[key])
+                                    .sortBy((_, key) => this.iMatricsFields[key].order,
                                 );
 
+                                const othersGroupedAndSortedNotInConfig = othersGrouped
+                                    .filter(item => !othersGroupedAndSortedByConfig.includes(item))
```

Comment:
It'd be cleaner to make a `function hasConfig(key) { return this.iMatricsFields[key] != null) }`. That way, you would not need to iterate the list in order to check the value(.includes does the iteration).

The lists will be small here, so it's just a nice to have. If the lists were big, it'd be a performance issue.